content/en/docs/ops/ambient/helm-installation/index.md

```diff
@@ -0,0 +1,175 @@
+{{< /text >}}
+
+## Configuration
+
+To view support configuration options and documentation, run:
+
+{{< text syntax=bash >}}
+$ helm show values istio/istiod
+{{< /text >}}
+
+## Verifying the Installation
+
+### Verifying the deployments status
+
+After installing all the components, you can check the helm deployment status:
+
+{{< text syntax=bash snip_id=show_components >}}
+$ helm list -n istio-system
+{{< /text >}}
+
+You can check the status of the pods deployed:
+
+{{< text syntax=bash snip_id=check_deployments >}}
+$ kubectl get pods -n istio-system
+{{< /text >}}
+
+### Verifying with the Sample Application
+
+After installing ambient with Helm, you can follow
+[Deploy the sample application](/docs/ops/ambient/getting-started/#bookinfo)
+guide to deploy the sample application and ingress gateways, and follow the
+[Adding your application to ambient](/docs/ops/ambient/getting-started/#addtoambient)
+to add your application to ambient.
+
+## Uninstall
+
+You can uninstall Istio and its components by uninstalling the charts
+installed above.
+
+1. List all the Istio charts installed in `istio-system` namespace:
+
+    {{< text syntax=bash >}}
+    $ helm ls -n istio-system
+    NAME       NAMESPACE    REVISION UPDATED         STATUS   CHART        APP VERSION
+    istio-base istio-system 1        ... ... ... ... deployed base-1.0.0   1.0.0
+    istiod     istio-system 1        ... ... ... ... deployed istiod-1.0.0 1.0.0
+    {{< /text >}}
+
+1. (Optional) Delete any Istio gateway chart installations:
+
+    {{< text syntax=bash >}}
+    $ helm delete istio-ingress -n istio-ingress
+    $ kubectl delete namespace istio-ingress
+    {{< /text >}}
+
+1. Delete Istio CNI chart:
+
+    {{< text syntax=bash snip_id=delete_cni >}}
+    $ helm delete istio-cni -n istio-system
+    {{< /text >}}
+
+1. Delete Istio ztunnel chart:
+
+    {{< text syntax=bash snip_id=delete_ztunnel >}}
+    $ helm delete ztunnel -n istio-system
+    {{< /text >}}
+
+1. Delete Istio discovery chart:
+
+    {{< text syntax=bash snip_id=delete_discovery >}}
+    $ helm delete istiod -n istio-system
+    {{< /text >}}
+
+1. Delete Istio base chart:
+
+    {{< tip >}}
+    By design, deleting a chart via Helm doesn't delete the installed Custom
+    Resource Definitions (CRDs) installed via the chart.
+    {{< /tip >}}
+
+    {{< text syntax=bash snip_id=delete_base >}}
+    $ helm delete istio-base -n istio-system
+    {{< /text >}}
+
+1. Delete the `istio-system` namespace:
+
+    {{< text syntax=bash snip_id=delete_system_namespace >}}
+    $ kubectl delete namespace istio-system
+    {{< /text >}}
+
+## Uninstall stable revision label resources
+
+If you decide to continue using the old control plane, instead of completing the update,
+you can uninstall the newer revision and its tag by first issuing
+`helm template istiod istio/istiod -s templates/revision-tags.yaml --set revisionTags={prod-canary} --set revision=canary -n istio-system | kubectl delete -f -`.
```

Comment:
```suggestion
you can uninstall the newer revision and its tag by first running:

{{< text syntax=bash >}}
$ helm template istiod istio/istiod -s templates/revision-tags.yaml --set revisionTags={prod-canary} --set revision=canary -n istio-system | kubectl delete -f -
{{< /text >}}

```

Reply:
Thanks for all comments, revised.

Reply:
This cannot be revised, otherwise we will violate some lint rules. This is actually copied from https://istio.io/latest/docs/setup/install/helm/#uninstall-stable-revision-label-resources.